File: C^C++/不高兴的津津（升级版）.cpp
//ÌâÄ¿Á´½Ó:"https://www.luogu.org/problemnew/show/P1534" 
#include<bits/stdc++.h>
using namespace std;
int main()
{
	int n;
	cin>>n;
	int SUM=0;
	int sum=0;
	for(int i=0;i<n;i++)
	{
		int a,b;
		cin>>a>>b;
		SUM=SUM+a+b-8;
		sum+=SUM;
	}
	cout<<sum;
	return 0;
}

File: JAVA/不高兴的津津（升级版）.java
//题目链接:"https://www.luogu.org/problemnew/show/P1534"
import java.util.*;

public class Main {
    public static void main(String[] args){
        Scanner s = new Scanner  (System.in);
        int n = s.nextInt();
        int sum=0,SUM=0;
        for(int i=0;i<n;i++){
            int a = s.nextInt(),b = s.nextInt();
            sum=sum+a+b-8;
            SUM+=sum;
        }
        System.out.print(SUM);
    }
}
